### PR TITLE
PaaS APIs and Tests are added.

### PR DESCRIPTION
**Pull request by nvthongswansea**

### CODE REVIEW

File: loadbalancer_test.go [hunk 1]
```diff
@@ -19,19 +20,19 @@ func setupTestClient() (*Client, *http.ServeMux) {
 	server := httptest.NewServer(mux)
 	config := Config{
 		APIUrl:     server.URL,
-		UserUUID:   "uuid",
-		APIToken:   "token",
+		UserUUID:   os.Getenv("USER_UUID"),
+		APIToken:   os.Getenv("API_TOKEN"),
 		HTTPClient: http.DefaultClient,
```
Comment (itakouna) [inline loadbalancer_test.go]: setupTestClient function should be moved to file `testcommon.go`.
No need ` os.Getenv("USER_UUID")` `os.Getenv("API_TOKEN")`, at the end it's dummy values to configure client.

Comment (itakouna) [reply loadbalancer_test.go]: A further improvement, setupTestClient function should return `server`, so we can use `defer  server.Close()` directly after calling `setupTestClient` in the tests.

File: loadbalancer_test.go [hunk 2]
```diff
@@ -54,29 +55,29 @@ func TestCreateLoadBalancer(t *testing.T) {
...
 	})
 	loadbalancer, err := client.GetLoadBalancer(loadBalancerID)
 	if err != nil {
 		t.Errorf("GetLoadBalancer returned error: %v", err)
 	}
 	assert.Equal(t, fmt.Sprintf("%v", expectedObject.Properties), fmt.Sprintf("%v", loadbalancer.Properties))
 }
-func TestGetLoadBalancerList(t *testing.T) {
+func TestClient_GetLoadBalancerList(t *testing.T) {
 	client, mux := setupTestClient()
```
Comment (itakouna) [inline loadbalancer_test.go]: This become like the following
```
server, client, mux := setupTestClient()
defer  server.Close()
```

**Comment by nvthongswansea** [comment]
All common functions and dummy variables used for testing are move to test common.go.
Server for testing now will be closed after finishing each unit test.

**Comment by itakouna** [comment]
I just noticed you forget to update `README.md`

**Pull request closed without merging**